ewatercycle/parameter_sets/__init__.py

```diff
@@ -0,0 +1,105 @@
+from itertools import chain
+from logging import getLogger
+from os import linesep
+from typing import Iterable
+
+from ewatercycle import CFG
+from . import _pcrglobwb, _lisflood, _wflow
+from .default import ParameterSet
+from ._example import ExampleParameterSet
+from ..config import SYSTEM_CONFIG, USER_HOME_CONFIG
+
+logger = getLogger(__name__)
+
+
+def _parse_parametersets():
+    parametersets = {}
+    for name, options in CFG["parameter_sets"].items():
+        parameterset = ParameterSet(name=name, **options)
+        parametersets[name] = parameterset
+
+    return parametersets
+
+
+def available_parameter_sets(target_model: str = None) -> Iterable[str]:
```

Comment:
`ewatercycle.parameter_sets.available_parameter_sets(target_model="lisflood")` currently returns a generator, which is not ideal for the public API (and not consistent with the example notebook). It would be nice if we can incorporate this example in one of our notebooks, and ideally it should return a list of parameterset names or a dict with them names as keys.

Reply:
Changed it to tuple of strings as that more closely matches to api notebook.